lib/db/schema/patch-020-021.sql

```diff
@@ -0,0 +1,35 @@
+CREATE PROCEDURE `sessionDevice_1` (
+  IN `inTokenId` BINARY(32)
+)
+BEGIN
+  SELECT
+    t.tokenData,
+    t.uid,
+    t.createdAt,
+    t.uaBrowser,
+    t.uaBrowserVersion,
+    t.uaOS,
+    t.uaOSVersion,
+    t.uaDeviceType,
+    t.lastAccessTime,
+    a.emailVerified,
+    a.email,
+    a.emailCode,
+    a.verifierSetAt,
+    a.locale,
+    a.createdAt AS accountCreatedAt,
+    d.id AS deviceId,
+    d.name AS deviceName,
+    d.type AS deviceType,
+    d.createdAt AS deviceCreatedAt,
+    d.callbackURL,
+    d.callbackPublicKey
+  FROM
+    sessionTokens t
+    LEFT JOIN accounts a ON t.uid = a.uid
+    LEFT JOIN devices d ON t.tokenId = d.sessionTokenId
+  WHERE
+    t.tokenId = inTokenId;
```

Comment:
@rfk :+1: 

Before:
```
+----+-------------+-------+-------+---------------+---------+---------+-------+------+-------------+
| id | select_type | table | type  | possible_keys | key     | key_len | ref   | rows | Extra       |
+----+-------------+-------+-------+---------------+---------+---------+-------+------+-------------+
|  1 | SIMPLE      | t     | const | PRIMARY       | PRIMARY | 32      | const |    1 | NULL        |
|  1 | SIMPLE      | a     | const | PRIMARY       | PRIMARY | 16      | const |    1 | NULL        |
|  1 | SIMPLE      | d     | ALL   | NULL          | NULL    | NULL    | NULL  |    1 | Using where |
+----+-------------+-------+-------+---------------+---------+---------+-------+------+-------------+
```

with `LEFT JOIN devices d ON (t.tokenId = d.sessionTokenId AND t.uid = d.uid)`:

```
+----+-------------+-------+-------+-----------------------------------+---------------------------+---------+-------------+------+-------+
| id | select_type | table | type  | possible_keys                     | key                       | key_len | ref         | rows | Extra |
+----+-------------+-------+-------+-----------------------------------+---------------------------+---------+-------------+------+-------+
|  1 | SIMPLE      | t     | const | PRIMARY                           | PRIMARY                   | 32      | const       |    1 | NULL  |
|  1 | SIMPLE      | a     | const | PRIMARY                           | PRIMARY                   | 16      | const       |    1 | NULL  |
|  1 | SIMPLE      | d     | const | PRIMARY,UQ_devices_sessionTokenId | UQ_devices_sessionTokenId | 49      | const,const |    1 | NULL  |
+----+-------------+-------+-------+-----------------------------------+---------------------------+---------+-------------+------+-------+
```